### DISCUSSION TITLE
Ability to open links inside the app *ANDROID*

### DISCUSSION THREAD

**Discussion by BrajBliss** [community/community#4496] [Mobile]
Just how many apps can open links directly inside the app (although using the browser) but in the app as a custom tab for better ease.

**Comment by fvdm** [comment]
This is already possible: Settings > External Links

**Reply by BrajBliss** [reply]
There is no such option ;-;

**Reply by fvdm** [reply]
I'm on v1.23.0

<img src="https://user-images.githubusercontent.com/93784/126657524-f1876b44-f4bc-42be-aef6-83abf50ac175.PNG" width="200">

**Reply by BrajBliss** [reply]
Android :)
